contracts/Everlong.sol

```diff
@@ -104,6 +100,12 @@ contract Everlong is IEverlong {
     /// @dev Decimals to use with asset.
     uint8 internal immutable _decimals;
 
+    /// @notice Target percentage of assets to leave uninvested.
+    uint256 public immutable targetIdleLiquidityPercentage;
+
+    /// @notice Maximum percentage of assets to leave uninvested.
+    uint256 public immutable maxIdleLiquidityPercentage;
+
     /// @dev Kind of everlong.
```

Comment:
Nit: One thing I noticed yesterday is that several variables that are now marked as public have @dev tags. Let’s update those to have @notice tags